net-p2p/freenet/freenet-0.7.5_p1488-r2.ebuild

```diff
@@ -0,0 +1,169 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+JAVA_PKG_IUSE="doc source"
+
+inherit epatch java-pkg-2 java-ant-2 systemd
```

Comment:
DeprecatedEclass: version 0.7.5_p1488-r2: uses deprecated eclass: epatch (migrate to eapply from EAPI 7)

Reply:
Let's see whether this package's maintainer is interested in fixing these QA issues :)